### ISSUE TITLE
multiword (contractions)

### ISSUE THREAD

**Issue by arademaker** [own-pt/cl-conllu#1]
http://universaldependencies.org/format.html

The read and write functions still do not deal with the multiword lines.

**Comment by arademaker** [comment]
See https://github.com/own-pt/bosque-UD/issues/23#issuecomment-266827081 how to encode the fusions and contractions ? @fcbr my idea would be to add one more slot in the sentence class for the list of contractions, what do you think ?

**Comment by arademaker** [comment]
BTW, @fcbr note that UD_Portuguese is not following the documentation, since the lines should contain 10 fields as all other tokens

**Comment by arademaker** [comment]
@GPPassos can you and João try next to solve this issue?

**Comment by fcbr** [comment]
That seems a good suggestion.

**Comment by arademaker** [comment]
I have simplified the code and tested it, closed by addd6fd2729903880903b951880549e478222ba2

**Comment by arademaker** [comment]
The final test was:

```
$ sbcl --eval '(ql:quickload :cl-conllu)' --eval '(in-package :cl-conllu)'  --eval '(write-conllu (append (read-conllu "pt-ud-dev.conllu") (read-conllu "pt-ud-test.conllu") (read-conllu "pt-ud-train.conllu")) "lixo.conllu")'
$ cat pt-ud-dev.conllu pt-ud-test.conllu pt-ud-train.conllu > lixo1.conllu
$ diff lixo.conllu lixo1.conllu
```
